packages/gatsby-remark-copy-linked-files/src/index.js

```diff
@@ -32,19 +32,23 @@ const newPath = (linkNode, destinationDir) => {
   return path.posix.join(process.cwd(), DEPLOY_DIR, newFileName(linkNode))
 }
 
-const newLinkURL = (linkNode, destinationDir) => {
-  if (destinationDir) {
-    return path.posix.join(`/`, destinationDir, newFileName(linkNode))
-  }
-  return path.posix.join(`/`, newFileName(linkNode))
+const newLinkURL = (linkNode, destinationDir, pathPrefix) => {
+  const linkPaths = [`/`, pathPrefix, destinationDir, newFileName(linkNode)]
+    .filter(function(lpath) {
+      if (lpath) return true
+      return false
+    })
+
+  return path.posix.join(...linkPaths)
 }
 
 module.exports = (
-  { files, markdownNode, markdownAST, getNode },
+  { files, markdownNode, markdownAST, pathPrefix, getNode },
   pluginOptions = {}
 ) => {
   const defaults = {
     ignoreFileExtensions: [`png`, `jpg`, `jpeg`, `bmp`, `tiff`],
+    pathPrefix,
```

Comment:
This isn't necessary since `pathPrefix` is a global option and plugins shouldn't have option for overriding it.

Reply:
Sorry for the delay, I had to do some investigation into what other packages were doing.

When I remove this it fails, what should I be doing here instead? It's not clear how I should be using `pathPrefix` for options. 

I was using [remark-images](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-remark-images/src/index.js#L16-L26) for reference originally. Is this incorrect as well?

Should I be using globals like [gatsby-link](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-link/src/index.js#L1-L10) instead?

Reply:
What I'm saying is just pass `pathPrefix` to `newLinkUrl` instead of making `pathPrefix` an option.